src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs

```diff
@@ -1592,21 +1593,42 @@ public void CrossCorlibSystemObjectReturnType_Script()
             //
             // In the original bug, Xamarin iOS, Android, and Mac Mobile profile corlibs were
             // realistic cross-compilation targets.
-            var compilation = CSharpCompilation.CreateScriptCompilation(
-                "submission-assembly",
-                references: new [] { MinAsyncCorlibRef },
+
+            Action<CSharpCompilation> AssertCompilationCorlib = compilation =>
```

Comment:
Apparently not. If I convert this to a local function, VS2015 cannot compile it.